forge-gui/res/cardsfolder/a/ashnods_altar.txt

```diff
@@ -4,4 +4,6 @@ Types:Artifact
 A:AB$ Mana | Cost$ Sac<1/Creature> | Produced$ C | Amount$ 2 | SpellDescription$ Add {C}{C}.
 SVar:NonStackingEffect:True
 AI:RemoveDeck:All
```

Comment:
hmmm, not sure this is needed anymore... seems like the AI would be able to do it, maybe with a Sac Pref line

Reply:
as it is, not sure how much DeckHas or Hints even do when you've got a RemoveDeck:All